pandas/tests/dtypes/test_common.py

```diff
@@ -391,6 +392,41 @@ def test_is_unsigned_integer_dtype(dtype):
 def test_is_not_unsigned_integer_dtype(dtype):
     assert not com.is_unsigned_integer_dtype(dtype)
 
+def test_all_type_comparison():
+    assert pd.api.types.pandas_dtype("int_") == "int_"
```

Comment:
pls paramterize instead of writing out like this

Reply:
I made the change you requested